# DP: Description: Optimizes the build for sparcv8
# DP: Author: Ben Collins
# DP: Upstream status: Debian-Specific
# DP: Status Details:
# DP: Date: June 17, 2003

Index: configure
===================================================================
--- configure.orig
+++ configure
@@ -1798,6 +1798,11 @@
   machine="powerpc"
 fi
 
+# Force v8
+if test "$machine" = sparc; then
+   machine="sparcv8"
+fi
+
 # Braindead PowerPC box with absolutely no FPU.
 case "$machine-$host_os" in
   powerpc*-*soft)
Index: configure.in
===================================================================
--- configure.in.orig
+++ configure.in
@@ -307,6 +307,11 @@
   machine="powerpc"
 fi
 
+# Force v8
+if test "$machine" = sparc; then
+   machine="sparcv8"
+fi
+
 # Braindead PowerPC box with absolutely no FPU.
 case "$machine-$host_os" in
   powerpc*-*soft)
